test/acceptance/integration/ac_mdw/__init__.py

```diff
@@ -1 +1,24 @@
-__author__ = 'Jon'
+# -*- coding: utf-8 -*-
```

Comment:
In general, ac_mdw/ files seems to be tests for the AC itself... It's that correct? Are they a requeriment for PEP test?